pom.xml

```diff
@@ -136,15 +157,9 @@
       </dependency>
 
       <dependency>
-        <groupId>com.oracle.wcc</groupId>
+         <groupId>oracle.ucm</groupId>
         <artifactId>ridc</artifactId>
-        <version>[11.1.1,12.0.0.0)</version>
-      </dependency>
-      <!-- TODO: should this be updated to: 4.4.1 for RIDC? -->
-      <dependency>
-        <groupId>commons-httpclient</groupId>
-        <artifactId>commons-httpclient</artifactId>
-        <version>3.1</version>
+         <version>12.2.2.0.0</version>
```

Comment:
I had the variable dependency, on ridc version, because we're still on 11G.  I don't want to restrict to WCC 12C and above yet.

Reply:
RIDC is backward compatible. It works just fine for older than 12c.